iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/relational/analyzer/StatementAnalyzer.java

```diff
@@ -362,8 +366,50 @@ protected Scope visitShowIndex(ShowIndex node, Optional<Scope> context) {
     }
 
     @Override
-    protected Scope visitUpdate(Update node, Optional<Scope> context) {
-      throw new SemanticException("Update statement is not supported yet.");
+    protected Scope visitUpdate(final Update node, final Optional<Scope> context) {
+      queryContext.setQueryType(QueryType.WRITE);
+      final TranslationMap translationMap = analyzeTraverseDevice(node, context, true);
```

Comment:
TranslationMap should be used in logical planner step, not in statement analyze step.